app.yaml

```diff
@@ -0,0 +1,12 @@
+name: "app"
+only-arches: [ i386 ]
```

Comment:
You need to limit the builders using flathub.json too.

Reply:
Fixed on upcoming commit

Reply:
Let's keep it unresolved until it's pushed out.

Reply:
I already have a `flathub.json` on latest commit though...

Reply:
That's fine, just remove `only-arches` from here and it's good.